### ISSUE TITLE
[TW-1154] start command should disallow other args

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1189]
_Paul Beckingham on 2009-08-16T15:02:28Z says:_ 

 Charles T. Yun reported that the following ambitious (and unsupported) command (a) doesn't work, and (b) doesn't tell you that it didn't work.

    t start 241 pri:l

The idea was to both start the task and lower its priority.  Sorry Charles.

The short term solution will be to generate an error.  Long term, this will come under the new command line filtering/parsing, but I'm not sure it's something we considered yet.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2009-08-16T15:02:28Z
Modified: 2014-02-09T02:37:22Z
```

**Comment by taskwarrior** [comment]
_t charles yun on 2009-08-21T08:50:21Z says:_ 

 > The short term solution will be to generate an error.  Long term, this will come under the new command line filtering/parsing, but I'm not sure it's something we considered yet.

howdy.  Thanks for getting this addressed in the 1.8.1 release.

I had a quick suggestion for whenever the next release happens.  The current error states that the start param does not allow for further modification.  It might be nice to further state "please enter the modifications individually in separate commands" or the like.  It just gives a bit of a hint as to what the user should do.

Congrats and thanks again on 1.8.1, 
*  tcy
